### PR TITLE
Firebase deployment and convert configuration/keycloak json to .env

### PR DESCRIPTION
**Pull request by pwei1018**
*Issue #:* [15500](https://github.com/bcgov/entity/issues/15500)

*Description of changes:*
1. Enable firebase deployment;
2. Convert configuration.json to .env;
3. Convert keycloak.json to .env;
4. Remove SbcSystemAlert and replace with banner-text;

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/App.vue [hunk 1]
```diff
@@ -210,6 +195,13 @@ export default class App extends Mixins(AuthMixin) {
     return process.env.ABOUT_TEXT
   }
 
+  /** Get banner text. */
+  private get bannerText (): string {
+    const bannerText: string = getFeatureFlag('banner-text')
+    // remove spaces so that " " becomes falsy
+    return bannerText?.trim() || null
+  }
+
```
Comment (cameron-eyds) [inline ppr-ui/src/App.vue]: Small heads up here, i've a PR open which reformats the whole `App.vue` script to composition api script setup (from a class component) as part of our prep for the migration to V3. 
There will certainly be some conflicts.

Comment (pwei1018) [reply ppr-ui/src/App.vue]: When do you plan to merge your PR? I can wait for it and solve all conflicts.

Comment (cameron-eyds) [reply ppr-ui/src/App.vue]: Hopefully early today!

Comment (cameron-eyds) [reply ppr-ui/src/App.vue]: If this PR needs to get in asap, i could also hold on my PR and resolve the conflicts. 
Whatever you prefer!

**Review by cameron-eyds — APPROVED** [review]

File: ppr-ui/src/App.vue (hunk 1 above)
Comment (pwei1018) [reply ppr-ui/src/App.vue]: I can wait for your PR. I need to talk to Doug to figure out some API key config as well.

**Review by dimak1 — APPROVED** [review]
Other than Cameron's comment, all looks good.

File: ppr-ui/src/App.vue (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/App.vue]: PR is merged, will be some minor conflicts with the version and App.vue. 

The code in the template should be a-ok, just the computed values are structured a bit different in the script setup for composition api. Lots of examples in App.vue though :) 
Lmk if there is any issues!

**Pull request merged**